code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -125,6 +130,9 @@
 	else
 		if(status)
 			baton_stun(L, user)
+			if(user.real_name == user.name && shitcurity) //only say it if you aren't disguised
+				user.say("[pick("STOP RESISTING ARREST", "YOU HAVE THE RIGHT TO REMAIN SILENT", "REACH FOR THE SKY", "DROP THE WEAPON", "IF YOU CAN'T DO THE TIME, DON'T DO THE CRIME", \

```

Comment:
"You have the right to remain silent." Would be the only one hear I wouldn't mind. But in regular case, and with a period.